### PR TITLE
Many improvements

### PR DESCRIPTION
**Pull request by aleksanb**
stuff

### CODE REVIEW

File: nin/frontend/app/scripts/controllers/main.js [hunk 1]
```diff
@@ -124,12 +124,13 @@
...
-            {name: 'Layer', click: function() {
+            {name: 'Node', click: function() {
               commands.pause();
-              var layerName = window.prompt("Enter a name for the layer:");
-              commands.generate('layer', layerName);
+              var layerName = window.prompt("Enter a name for the node:");
+              commands.generate('node', layerName);
             }},
-            {name: 'Shader', click: function() {
+            // TODO: Currently broken, hide untill fixed
```
Comment (sigvef) [inline nin/frontend/app/scripts/controllers/main.js]: Might as well delete entire block?

File: nin/backend/serve.js [hunk 2]
```diff
@@ -94,12 +94,14 @@ var serve = function(projectPath, shouldRunHeadlessly) {
         var directoryPrecedence = {'lib': 0, 'src': 1, 'res': 2};
         var directoryAScore = directoryPrecedence[a.slice(0, 3)];
         var directoryBScore = directoryPrecedence[b.slice(0, 3)];
+
         if(directoryAScore == directoryBScore) {
           return a > b;
         }
-        return directoryAScore > directoryBScore;
 
+        return directoryAScore > directoryBScore;
```
Comment (sigvef) [inline nin/backend/serve.js]: What changed here? o.O

**Comment by aleksanb** [comment]
Ping. Removed the commented out feature.

**Comment by sigvef** [comment]
Make doesn't work:

image

Perhaps we should wait for the upcoming webpack migration to land, and then rebase over that?

**Comment by stianjensen** [comment]
Superseded by #268

**Pull request closed without merging**